### ISSUE TITLE
Improve Swimming

### ISSUE THREAD

**Issue by ComputerSmoke** [Inspirateur/riverbed#40]
Controls and physics should be affected when a player is in water. Additionally, there should be a visual indication when underwater. This could be something like volumetric water or an effect on the screen.

**Comment by Inspirateur** [comment]
Yeah i'll work on this for sure at some point :p Ty for putting up the issue
